src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -50,7 +60,29 @@ export class RelayFeeCalculator {
       this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
       "capitalCostsPercent must be between 0 and 100 percent"
     );
+    this.capitalCostsConfig = config.capitalCostsConfig ? JSON.parse(config.capitalCostsConfig) : {};
```

Comment:
you shouldnt need to parse this now that its typed

Reply:
agreed